### PR TITLE
Update variables in run_post_receive to fit the meaning

### PR DESCRIPTION
**Pull request by timfanda35**
#81

### CODE REVIEW

File: pyas2/utils.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ def run_post_receive(message, full_filename):
         variables = {
             "filename": os.path.basename(full_filename),
             "fullfilename": full_filename,
-            "sender": message.organization.as2_name,
-            "receiver": message.partner.as2_name,
+            "sender": message.partner.as2_name,
```
Comment (abhishek-ram) [inline pyas2/utils.py:43]: should this not also need a test change

Comment (timfanda35) [reply pyas2/utils.py:43]: It should be, I will add the test.

File: setup.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "pytest-black==0.3.12",
     "black==22.6.0",
     "django-environ==0.4.5",
+    "pyflakes==2.4.0",
```
Comment (abhishek-ram) [inline setup.py:27]: where is this used?

Comment (timfanda35) [reply setup.py:27]: pyflakes is used by pylama. I got errors about it when running pytest.

```
AttributeError: module 'pyflakes.messages' has no attribute 'RedefinedInListComp'
```

I found the issue at https://github.com/klen/pylama/issues/224. Follow the discussion to pin the pyflakes version and 
run pytest successfully.

**Review by abhishek-ram — APPROVED** [review]

**Pull request merged**